.github/workflows/label.yml

```
@@ -83,3 +87,15 @@ jobs:
           - [good first issue](https://github.com/ansys/pyansys-geometry/pulls?q=label%3Agood+first+issue)
           - [maintenance](https://github.com/ansys/pyansys-geometry/pulls?q=label%3Amaintenance+)
           - [release](https://github.com/ansys/pyansys-geometry/pulls?q=label%3Arelease+)
+
+  changelog-fragment:
+    name: "Create changelog fragment"
+    needs: [labeler]
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: ansys/actions/doc-changelog@feat/changelog-action
```

```suggestion
    - uses: ansys/actions/doc-changelog@main
```

We have to use main - just letting this comment for whenever the action is merged.